lib/src/test/java/org/team100/lib/config/SimulatedCameraTest.java

```diff
@@ -0,0 +1,28 @@
+package org.team100.lib.config;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.ArrayList;
+import java.util.Optional;
+
+import org.junit.jupiter.api.Test;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation3d;
+import edu.wpi.first.math.geometry.Transform3d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.math.geometry.Translation3d;
+
+public class SimulatedCameraTest {
```

Comment:
this test seems like a very simple case: no camera offset and no robot offset.  you should add some non-trivial cases to verify the function of the offsets.